tests/Scale.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'helpers/perf_run_helper'
+
+test_name 'Scale'
+
+teardown do
+  # TODO: remove?
+  # perf_teardown
+end
+
+# TODO: remove?
```

Comment:
yeah remove.  We have other references for this code should we want it...

Reply:
Removed